cvat/apps/engine/models.py

```diff
@@ -280,9 +280,9 @@ class FloatArrayField(models.TextField):
     separator = ","
 
     def from_db_value(self, value, expression, connection):
-            if value is None:
-                return value
-            return [float(v) for v in value.split(self.separator)]
+        if value is None or not value:
```

Comment:
Does the code below work?
```suggestion
        if not value:
```